utils/EventRepository.js

```diff
@@ -47,6 +48,8 @@ class EventRepository {
     this.url = "";
     this.state = "READY";
     this.batchSize = 0;
+    this.useBeacon = false;
+    this.beaconQueue = beaconQueue;
```

Comment:
For Ln 63, why is `payloadQueue` initialized by default?

Reply:
This is for xhr, haven’t changed anything.

Reply:
We're supposed to change the implementation to initialize either XHR or Beacon based on the load options.

Reply:
done